src/checkFirebaseSDKVersion.ts

```diff
@@ -0,0 +1,96 @@
+import * as _ from "lodash";
```

Comment:
Should we rename this file to checkFunctionsSDKVersion now?

Reply:
I want to leave this file as is, since there may be other checks for other SDKs in the future. And this is a general pre-deployment step for "firebase deploy" (in commands/deploy.js), not exclusive to functions.